### ISSUE TITLE
Exacerbated by pulmonary vasodilators

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11268]
**Preferred term label:**
Exacerbated by pulmonary vasodilators

**Synonyms**
Symptoms and vital signs worsen with initiation of pulmonary vasodilator medications

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C5401108 is similar, but doesn't specify whether the effect is good or bad

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pulmonary venoocclusive disease
Pulmonary hypertension
Pulmonary capillary hemangiomatosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Note that I added a few examples because I assume it is the vasodilation rather than the specific class?

Adding new term: 
- ID: HP:6001413
- label: Exacerbated by pulmonary vasodilator exposure
- parents: [Aggravated by exposure to medication (HP:4000176)](https://hpo.jax.org/app/browse/term/HP:4000176)
- pmid: 
- def: Applied to a sign or symptom that is worsened by treatment with a pulmonary vasodilator medication such as prostacyclin analogs, endothelin receptor antagonists, and phosphodiesterase-5 (PDE5) inhibitors.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11268
- synonyms: Exacerbated by pulmonary vasodilators(NONE)
